### ISSUE TITLE
Difference between Network id and ChainId

### ISSUE THREAD

**Issue by ulrichm69** [javahippie/geth-dev#40]
Hi 
Very nice setup!

When looking at "Building a local Ethereum network with Docker and Geth"
your genesis.json says  "chainId": 500,
but in your github repository it says chainId": 326432352,
Also in github repository
the node and miner dockerfiles says --networkid="500"
Shouldn´t these be the same 326432352?

**Comment by kyriediculous** [comment]
Yes either they should be equal or omitted. Will fix this in next PR.

**Comment by javahippie** [comment]
@kyriediculous I added you to the project as a contributor, are you able to approve pull requests?

**Comment by ulrichm69** [comment]
I would like to approve when the repository works out the box.

So far I had to manually update some xenial stuff and so far I cant get akk three miners up and running, because
changing the addresses of the different miners to separate addresses conflicts with the use of the same dockerfile where address/keys are imported from the same file
also the netstats localhost:3000 responds with connection refused, all other ports works fine
Then I need to figure out how to add additional addresses´ keystore files

So I have a lot to fix before it will work as intended.

Best Regards

Ulrich Metz

Fra: Tim Zöller <notifications@github.com>
Sendt: 15. november 2018 13:47
Til: javahippie/geth-dev <geth-dev@noreply.github.com>
Cc: Ulrich Metz <um@hornskovvindberg.dk>; Author <author@noreply.github.com>
Emne: Re: [javahippie/geth-dev] Difference between Network id and ChainId (#40)


@kyriediculous<https://github.com/kyriediculous> I added you to the project as a contributor, are you able to approve pull requests?
